src/languages/lib/ecmascript.js

```diff
@@ -1,4 +1,4 @@
-export const IDENT_RE = '[A-Za-z$_][0-9A-Za-z$_]*';
+export const IDENT_RE = '[A-Za-z$_][0-9A-Za-z$_]*\\??';
```

Comment:
Is `?` truly a valid ECMAScript identifier, or just in TS?

Reply:
Youre right , ive managed to scope the new fix to just ts in the new commit